modules/i3s/src/lib/parsers/parse-zip/buffer-file-provider.ts

```diff
@@ -0,0 +1,55 @@
+import {FileProvider} from './file-provider';
+
+/**
+ * Provides file data using DataView
+ */
+export class BufferFileProvider implements FileProvider {
```

Comment:
If this provides a file using DataView, why is it not called `DataViewFileProvider`? `BufferFileProvider` seems to suggest the Node.js `Buffer` class.